### ISSUE TITLE
Triggers eat mud output but status not shown

### ISSUE THREAD

**Issue by EddyBuhler** [missionz3r0/Dartmud-Lua-Scripts#21]
The dartmudlet triggers "eat"/suppress the trigger output, for example the auto conc messages, but there is no display anywhere on what the updated conc value is. I put additional echo lines in the triggers as a workaround, but should there be a status display somewhere that has all this info? If yes, it doesn't seem to be working, at least with Mudlet 4.13.1 on Linux.

**Comment by missionz3r0** [comment]
There should be a few windows that the messages are diverted to. Most of which happens in Scripts/UI.lua

Conc for instance happens [here](https://github.com/missionz3r0/Dartmud-Lua-Scipts/blob/95e0b56c5d82a7db6f7ce1f460fc8f5a835898e2/Scripts/UI.lua#L553)

If you're not seeing those additional windows then there could have been a breaking chance on mudlet. I know the mudlet version I last used was 4.1.

And it does look like they made some changes to Geysor, the Mudlet mini-window manager, in [4.10](https://github.com/Mudlet/Mudlet/releases/tag/Mudlet-4.10.0)

Getting any errors popping up whether on start up, or in the scripts console?

**Comment by EddyBuhler** [comment]
I've actually had some installation troubles. In the end, I ended up cloning the repo, then packaging it up into a .zip myself to actually have the "dartmudlet" package show up in the package browser.

I did get an error message on install that it couldn't find ~/.config/mudlet/profiles/DartMUD/dartmudlet/Personal_Scripts, so I created that and /resetup. Even before the /resetup, the additional windows didn't all seem to be working, though. Date is shown, who works, skillups are shown, but most of the "tabs" on the right are empty. I can upload a screenshot if you like.

One thing: In UI.lua:643, shouldn't it be setup() instead of resetup()? Similar case in Vote_Timers.lua.

This is what my UI looks like: https://i.postimg.cc/yBm1nqMt/image.png

I corrected both resetup() functions and did a /resetup after saving a backup copy of my dartmud db file, but the UI is still broken as in the screenshot.

**Comment by missionz3r0** [comment]
Yeah, you're right on that. Those should be setup rather than resetup. There was a pull request that I had thought fixed all of that, but perhaps not.

I'll dig through a bit to see about fixing the stragglers. Then get a new package made that you should be able to install from.

This is all assuming that something else isn't messed up.

**Comment by EddyBuhler** [comment]
I fixed those two spots locally and did a /resetup, but no change. /unsetup also doesn't remove the various windows. Something else definitely isn't working right.
